screengrab/spec/runner_spec.rb

```diff
@@ -8,12 +8,60 @@
     @runner = Screengrab::Runner.new(mock_executor, config, mock_android_environment)
   end
 
-  def set_mock_adb_response(command, mock_response)
+  def mock_adb_response_for_command(command, mock_response)
```

Comment:
Again, rubocop complains that I can't have a method with the word 'set' in it that's a setter. (even though that existed before this commit)